app/uk/gov/hmrc/transitmovementsrouter/connectors/PushNotificationsConnector.scala

```diff
@@ -92,4 +105,43 @@ class PushNotificationsConnectorImpl @Inject() (httpClientV2: HttpClientV2, appC
       }
     }
 
+  override def postForLargeMessages(movementId: MovementId, messageId: MessageId, messageType: MessageType, objectStoreURI: ObjectStoreURI)(implicit
```

Comment:
@liandra-giacomine we need to pass object-store uri headers in case of large message that is the requirement. No need to pass the message type as part of header here. So yeah better to manage with one method instead of duplicating the code.
@olakunlekuye2  I think you can follow ``` https://github.com/hmrc/common-transit-convention-traders/blob/main/app/v2/connectors/PersistenceConnector.scala postMovement ``` implementation.

Reply:
@drnaylor-oc if you say so! :)

Reply:
> we need to pass object-store uri headers in case of large message that is the requirement

When we need it. 

In this case, we don't need to pass the object-store URI here because whenever we have a large message, it's always going to be larger than the PPNS limit. So instead, save the hassle, just make body in the original endpoint optional, and if you have no body, just send a notification as if it's larger than 80kB.

Reply:
ohk. I thought we need to send object-store-url as part of notification to traders in case of large messages.

Reply:
No -- they can't even access the file in object store so it's useless to them. They will get their messages via the GET endpoint.